source/operations/monitoring/collect-minio-metrics-using-prometheus.rst

```diff
@@ -32,315 +32,136 @@ The procedure on this page documents the following:
 
-     For Prometheus deployments external to the cluster, you must specify an ingress or load balancer endpoint configured to route connections to and from the MinIO Tenant.
+The command returns output similar to the following:
 
-2) Restart Prometheus with the Updated Configuration
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-Append the desired ``scrape_configs`` job generated in the previous step to the configuration file:
-
-.. tab-set::
-
-   .. tab-item:: Cluster
-
-      Cluster metrics aggregate node-level metrics and, where appropriate, attach labels to metrics for the originating node.
-
-      .. code-block:: yaml
-         :class: copyable
-      
-         global:
-            scrape_interval: 60s
-         
-         scrape_configs:
-            - job_name: minio-job
-              bearer_token: TOKEN
-              metrics_path: /minio/v2/metrics/cluster
-              scheme: https
-              static_configs:
-              - targets: [minio.example.net]
-
-
-   .. tab-item:: Nodes
-
-      Node metrics are specific for node-level monitoring. You need to list all MinIO nodes for this configuration.
-
-      .. code-block:: yaml
-         :class: copyable
-      
-         global:
-            scrape_interval: 60s
-         
-         scrape_configs:
-            - job_name: minio-job-node
-              bearer_token: TOKEN
-              metrics_path: /minio/v2/metrics/node
-              scheme: https
-              static_configs:
-              - targets: [minio-1.example.net, minio-2.example.net, minio-N.example.net]
-
-	      
-   .. tab-item:: Bucket
-
-      .. code-block:: yaml
-         :class: copyable
-      
-         global:
-            scrape_interval: 60s
-         
-         scrape_configs:
-            - job_name: minio-job-bucket
-              bearer_token: TOKEN
-              metrics_path: /minio/v2/metrics/bucket
-              scheme: https
-              static_configs:
-              - targets: [minio.example.net]
-
-   .. tab-item:: Resource
-
-      .. code-block:: yaml
-         :class: copyable
-
-         global:
-            scrape_interval: 60s
-
-         scrape_configs:
-            - job_name: minio-job-resource
-              bearer_token: TOKEN
-              metrics_path: /minio/v2/metrics/resource
-              scheme: https
-              static_configs:
-              - targets: [minio.example.net]
-
-Start the Prometheus cluster using the configuration file:
+.. code-block:: yaml
+   :class: copyable
 
-.. code-block:: shell
+   scrape_configs:
+   - job_name: minio-job
+     bearer_token: TOKEN
+     metrics_path: /minio/metrics/v3
+     scheme: https
+     static_configs:
+     - targets: [minio.example.net]
+
+To scrape multiple types of metrics, run :mc-cmd:`mc admin prometheus generate --api-version v3 <mc admin prometheus generate --api-version>` for each type and add the ``job_name`` section to the ``scrape_configs`` in your Prometheus configuration.
```

Comment:
I guess this is how it works? I found examples in the prom docs that have multiple `job_name` sections.

Also, do we call parts of yaml files "sections?"

Reply:
I think they are nodes officially, but that doesn't seem very widely known or user friendly.

Reply:
Also, nodes is a conflict with other terms for our product.
So, `section` works well enough, I think?